.github/workflows/e2e.yaml

```diff
@@ -76,4 +84,16 @@ jobs:
           sleep 20
           echo ">>> Run Kyverno e2e test"
           make test-e2e
+          kubectl delete -f ${GITHUB_WORKSPACE}/definitions/install.yaml
 
+      - name: Update chart values
+        run: |
+          sed -i -e "s|nameOverride:.*|nameOverride: kyverno|g" charts/kyverno/values.yaml
+          sed -i -e "s|fullnameOverride:.*|fullnameOverride: kyverno|g" charts/kyverno/values.yaml
+          sed -i -e "s|namespace:.*|namespace: kyverno|g" charts/kyverno/values.yaml
+          sed -i -e "s|tag:.*|tag: $(git describe --always --tags)|g" charts/kyverno/values.yaml
+
+      - name: Run chart-testing (install)
+        run: |
+          kubectl create namespace kyverno
+          ct install --target-branch=main --namespace=kyverno
```

Comment:
Do we need to verify Kyverno is installed successfully?

Reply:
That's handled by the file added at `charts/kyverno/templates/tests/test.yaml`.  That file is deployed after the Helm chart as a test, I believe the way it works is anything inside the `tests` folder is used for sanity checking the chart when using the `ct install` command from chart-testing.  The test YALM that is deployed checks the liveness probe for Kyverno through the deployed service.

Reply:
Got it!